server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/ImportEmlRoutesTest.java

```diff
@@ -0,0 +1,178 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.webadmin.routes;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.inmemory.InMemoryId;
+import org.apache.james.mailbox.inmemory.manager.InMemoryIntegrationResources;
+import org.apache.james.mailbox.model.FetchGroupImpl;
+import org.apache.james.mailbox.model.MailboxId;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mailbox.model.MessageRange;
+import org.apache.james.mailbox.store.StoreMailboxManager;
+import org.apache.james.webadmin.WebAdminServer;
+import org.apache.james.webadmin.WebAdminUtils;
+import org.apache.james.webadmin.service.ImportEmlService;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.github.fge.lambdas.Throwing;
+
+import io.restassured.RestAssured;
+import io.restassured.filter.log.LogDetail;
+
+public class ImportEmlRoutesTest {
```

Comment:
please add : 
* a test with a big file that is supposed to work
* a test with a too big file
* a test with a invalid file format

Reply:
IMO only two last requirements should be added. If the routes can handle an extremely big file, they definitely can handle big file, too.

Reply:
IMO only the last requirement should be added. If the routes can handle an extremely big file, they definitely can handle big file, too.
File format should be eml.

Reply:
If you don't have a test, you can't know whether the application will crash, if there's a way to configure the limit, what the limit is, at which moment you are rejecting a file above the limit, etc.
So, please add those tests.

Reply:
The ticket only requires the route to import the body of an EML file, it does not require format validation.